cli/pkg/release/gbm.go

```diff
@@ -203,20 +206,124 @@ func CreateGbmPR(version, dir string) (gh.PullRequest, error) {
 }
 
 func renderGbmPrBody(version string, pr *gh.PullRequest) error {
+	// TODO - replace "" with dir variable
+	cl := getChangeLog("", pr)
+	rn := getReleaseNotes("", pr)
+
+	rc, err := CollectReleaseChanges(version, cl, rn)
+
+	if err != nil {
+		console.Error(err)
+	}
+
+	rfs := []gh.RepoFilter{
+		gh.BuildRepoFilter("gutenberg", "is:open", "is:pr", `label:"Mobile App - i.e. Android or iOS"`, fmt.Sprintf("v%s in:title", version)),
+		gh.BuildRepoFilter("WordPress-Android", "is:open", "is:pr", version+" in:title"),
+		gh.BuildRepoFilter("WordPress-iOS", "is:open", "is:pr", version+" in:title"),
+	}	
+
+	synced, err := gh.FindGbmSyncedPrs(*pr, rfs)
+	if err != nil {
+		console.Error(err)
+	}
+	
+	prs := []gh.PullRequest{}
+	for _, s := range synced {
+		prs = append(prs, s.Items...)
+	}
+
 	t := render.Template{
 		Path: "templates/release/gbm_pr_body.md",
 		Data: struct {
 			Version  string
 			GbmPrUrl string
+			Changes    []ReleaseChanges
+			RelatedPRs []gh.PullRequest
 		}{
-			Version: version,
+			Version:    version,
+			Changes:    rc,
+			RelatedPRs: prs,
 		},
 	}
 
 	body, err := render.Render(t)
 	if err != nil {
 		return err
 	}
+
 	pr.Body = body
 	return nil
 }
+
+func getChangeLog(dir string, gbmPr *gh.PullRequest) []byte {
+	var buff io.ReadCloser
+	cl := []byte{}
+
+	if dir == "" {
+		console.Warn("not implemented")
+
+		// TODO: find the best way to get the gbPr
+
+		// org, _ := repo.GetOrg("gutenberg")
+		// endpoint := fmt.Sprintf("https://raw.githubusercontent.com/%s/gutenberg/%s/packages/react-native-editor/CHANGELOG.md", org, gbPr.Head.Sha)
```

Comment:
We should determine the cleanest way to get `gbPr.Head.Sha` here, or refactor this if it's not needed.

Reply:
The `gh` search should work here. Something like what we are doing to find the GBM pr for the integration PRs
here: https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/trunk/cli/pkg/release/integrate.go#L144-L146

Reply:
That search might not actually work _as-is_ . Searching GH for PRs does not return all of the PR info. We have to make another request once we know the PR number.
I made some improvements to PR searching [here](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/pull/185/files#diff-7c6debd75d355092d930ab11518508e315ce45190d4bd377124d828eaa70d908)